src/client/app/effects/github.ts

```diff
@@ -134,12 +141,21 @@ export class GitHubEffects {
                 new Snippet.UpdateInfoAction({ id: snippetId, gist: gist.id, gistOwnerId: gist.owner.login })])
         )
         .catch(exception => {
+            if (exception instanceof Error) {
+                exception = JSON.parse(exception.message);
```

Comment:
Not sure if that's what Dan meant, but I think you should throw your own instance of an error.
Looks like we do already have one, actually:  `export class PlaygroundError extends CustomError {`.  Please use that one instead.  Any exception is an instance of `Error` so as is, your `if` check doesn't do much.